packaging/skia-perf-regression-copy-surface-as-draw.patch

```diff
@@ -0,0 +1,1199 @@
+diff --git a/src/gpu/GrDrawTarget.cpp b/src/gpu/GrDrawTarget.cpp
```

Comment:
Sure I can put a comment. I'll try to backport it to beta and stable if it makes sense and if possible.